code/datums/controllers/action_controls.dm

```diff
@@ -1009,31 +1007,47 @@ var/datum/action_controller/actions
 			if(item == source.equipped() && !I)
 				if(target.can_equip(item, slot))
 					logTheThing(LOG_COMBAT, source, "successfully puts \an [item] on [constructTarget(target,"combat")] at at [log_loc(target)].")
-					for(var/mob/O in AIviewers(owner))
-						O.show_message("<span class='alert'><B>[source] puts [item] on [target]!</B></span>", 1)
+					source.visible_message("<span class='alert'><B>[source] puts [item] on [target]!</B></span>", TRUE)
 					source.u_equip(item)
 					if(QDELETED(item))
 						return
 					target.force_equip(item, slot)
 					target.update_inv()
 		else if (I) //Wire: Fix for Cannot execute null.handle other remove().
+			// Picking a clowns pockets is likely to give you a stupid item instead
+			if (target.job == "Clown" && (slot == SLOT_L_STORE) || (slot == SLOT_R_STORE))
```

Comment:
As written this would work for any job with right pocket.
```suggestion
			if (target.job == "Clown" && (slot == SLOT_L_STORE || slot == SLOT_R_STORE))
```